### PR TITLE
Start work on example website

### PR DESCRIPTION
**Pull request by SaraVieira**
Its a lot, let me know your thoughts

CleanShot 2022-01-29 at 18 59 06@2x

### CODE REVIEW

**Comment by SaraVieira** [comment]
I have TS error and need to a description

**Comment by SaraVieira** [comment]
Feel free to try other gradients btw
I was using this: 
https://www.designgradients.com/?ref=producthunt
https://uigradients.com/

File: example/src/main.tsx [hunk 1]
```diff
@@ -1,75 +1,179 @@
...
+      <div className="sm:p-12 p-0 sm:static absolute sm:w-auto w-100vw left-0 max-w-full bg-slate-900 max-h-screen overflow-auto sm:text-base text-sm">
+        <div>
+          <div className="hidden sm:block ">
+            <nav className="flex" aria-label="Tabs">
+              {tabs.map((tab, idx) => (
+                <button
+                  type="button"
+                  key={tab.name}
+                  onClick={tab.onClick}
+                  className={classNames(
```
Comment (jfranciscosousa) [inline example/src/main.tsx:147]: Isn't object notation more readable here (imho)? Not a big deal though

**Comment by jfranciscosousa** [comment]
Looks pretty, where is the TS error though? XD

File: example/src/main.tsx (hunk 1 above)
Comment (SaraVieira) [reply example/src/main.tsx:147]: Does classnames support that?

File: example/src/main.tsx [hunk 2]
```diff
@@ -1,75 +1,179 @@
...
+      return `npm install ${packageName}`;
+    case 'pnpm':
+      return `pnpm install ${packageName}`;
+    case 'yarn':
+      return `yarn add ${packageName}`;
+  }
+};
 
 const App = () => {
+  const [packageManager, setPackageManager] = useState<PackageManagers>('npm');
```
Comment (SaraVieira) [inline example/src/main.tsx:44]: Here is the TS error

Comment (exqir) [reply example/src/main.tsx:44]: I guess this need to be `PackageManagers.NPM` or changing PackageMangers to `type PackageManagers = 'npm' | 'pnpm' | 'yarn'`

Comment (SaraVieira) [reply example/src/main.tsx]: You are super right, I keep forgetting you can use TS enums as actual values. Ended up making use of `PackageManagers.NPM` and its way cleaner now and has no random strings

Thank you

**Comment by SaraVieira** [comment]
We have a deployment: https://react-url-modal-ipkmok6yn-remotecom.vercel.app/

File: example/src/main.tsx (hunk 2 above)
Comment (exqir) [reply example/src/main.tsx]: Happy I could help ☺️ 

Looks pretty nice 🎉

**Comment by SaraVieira** [comment]
Updates:
CleanShot 2022-01-31 at 22 01 34@2x


This PR now also adds support for rendering a modal in a portal

@jfranciscosousa Can I get a pair of eyes in here?

File: example/src/main.tsx (hunk 2 above)
Comment (SaraVieira) [reply example/src/main.tsx]: Danke :D

File: example/src/main.tsx (hunk 1 above)
Comment (jfranciscosousa) [reply example/src/main.tsx:158]: Yes

```js
classNames('px-4 py-3 font-medium text-sm flex-grow', {
   'rounded-l-md': idx === 0,
   'rounded-r-md', idx === tabs.length - 1,
   'border-0 border-r-2 border-r-slate-900': idx !== tabs.length - 1,
   'bg-indigo-800 text-slate-100': !!tab.current,
   'text-slate-400 hover:text-slate-100 hover:bg-indigo-800 bg-slate-800': !tab.current
})
```

I think that works

Comment (SaraVieira) [reply example/src/main.tsx:158]: That does not work :(

**Review by jfranciscosousa — APPROVED** [review]

**Pull request merged**